presto-product-tests/conf/presto/etc/catalog/iceberg.properties

```diff
@@ -0,0 +1,2 @@
+connector.name=iceberg
+hive.metastore.uri=thrift://hadoop-master:9083
```

Comment:
@pratyakshsharma Can you try adding the below two configs as well to this properties file

```
hive.metastore.thrift.client.socks-proxy=hadoop-master:1180
hive.config.resources=/docker/volumes/conf/presto/etc/hive-default-fs-site.xml
```

Reply:
this did not help.

Reply:
I can probably remove these 2 configs now since they are not needed.